### PR TITLE
Integrate Apple and Schools studies into Reportable Conditions

### PR DESCRIPTION
**Pull request by kschwabefry**
Includes commits:
- shipping.reportable_condition_v1: add collection set
- reportable_conditions: send to channels based on collection set
- reportable_conditions: add slack integrations for apple and schools

Before deploy:
- [x] tag schema

### CODE REVIEW

**Comment by kschwabefry** [comment]
[Update reportable_conditions.py to send Apple and Schools positives to specific Slack channels](https://trello.com/c/4TcMuFGv/970-update-reportableconditionspy-to-send-apple-and-schools-positives-to-specific-slack-channels)

File: lib/seattleflu/id3c/cli/command/reportable_conditions.py [hunk 1]
```diff
@@ -107,10 +105,7 @@ def webhook(suffix):
                 responses = {'ncov-reporting': send_slack_post_request(record, slack_webhooks['ncov-reporting'])}
 
                 # Also send Childcare specific results to the #ncov-reporting-childcare channel
-                if (record.site in childcare['sites'] or
-                    record.sample_origin == childcare['sample_origin']  or
-                    record.swab_site in childcare['swab_sites']):
-
+                if (record.collection_set_name in childcare['collection_sets']):
```
Comment (jccraft-uw) [inline lib/seattleflu/id3c/cli/command/reportable_conditions.py]: Using collection set name is so much cleaner !!

File: lib/seattleflu/id3c/cli/command/reportable_conditions.py [hunk 2]
```diff
@@ -104,10 +123,11 @@ def webhook(suffix):
...
 
-                # Also send Childcare specific results to the #ncov-reporting-childcare channel
-                if (record.collection_set_name in childcare['collection_sets']):
-                    responses['ncov-reporting-childcare'] = send_slack_post_request(
-                        record, slack_webhooks['ncov-reporting-childcare'])
+                # Also send study-specific results to their respective channels
+                for project in projects:
+                    if (record.collection_set_name in project['collection_sets']):
+                        responses[project['slack_channel_name']] = send_slack_post_request(
+                            record, project['slack_webhook'])
```
Comment (jccraft-uw) [inline lib/seattleflu/id3c/cli/command/reportable_conditions.py:130]: If we never expect a collection set name to be used to send to more than one custom channel, then you could break here to save a little time.

**Review by jccraft-uw — APPROVED** [review]

**Review by joverlee521 — APPROVED** [review]
This looks good to me! Thanks for making the change to using identifier sets to split slack channels, it's much cleaner 🌟

**Comment by kschwabefry** [comment]
Deployed.

**Pull request merged**